tests/Probe.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from '@playwright/test';
+import { visitStudy, checkForScreenshot, screenShotPaths, simulateClicksOnElement } from './utils';
+
+test.beforeEach(async ({ page }) => {
+  const studyInstanceUID = '1.3.6.1.4.1.25403.345050719074.3824.20170125095438.5';
+  const mode = 'Basic Viewer';
+  await visitStudy(page, studyInstanceUID, mode);
+});
+
+test('should display the probe tool', async ({ page }) => {
+  await checkForScreenshot(page, page, screenShotPaths.probe.viewportLoaded);
```

Comment:
I don't understand why we start by checking the screenshot, that should be a separate test to check loading

Reply:
I answered above